src/exec/utils/utils_env.c

```diff
@@ -107,8 +107,9 @@ int	ft_setenv(t_data *data, t_env **env, char *var, char *value)
 				return (0);
 			free(current->value);
 			current->value = ft_strdup(value);
+			free(value);
```

Comment:
@aascedu jsp si ca va pas double free ca, j'ai test mais j'ai pas l'impression. si tu trouves un truc redis moi je ferais autrement

Reply:
fixed with ee5097e and 931c413